soql-server-pg/docker/ship.d/run

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -ev
+
+/bin/env_parse ${SERVER_CONFIG}.j2
+
+exec su socrata -c '/usr/bin/java \
+    -Xmx${JAVA_XMX} \
+    -Xms${JAVA_XMX} \
+    -Dconfig.file=${SERVER_CONFIG} \
```

Comment:
A good question.  I really would prefer to not specify the path under the DRY principle: we said what our context is once (by definition, that is what WORKDIR does), so why repeat it.

I'm confused though because it seems to work fine here and in almost every other project but didn't in the secondary-watcher.  Or I was just confused there.

I'd prefer to stick with not specifying it in general where it works, and go back and figure out why it didn't seem to work in secondary-watcher when we have time.